test/strings/basic.jl

```diff
@@ -552,3 +552,75 @@ end
     @test_throws ParseError parse("\"\\.\"")
     @test_throws ParseError parse("\'\\.\'")
 end
+
+# prevind and nextind tests
+
+let
```

Comment:
Sorry for the repeated change requests, but this file has just been ported to using testsets. Could you change the new tests to use `@testset` (and remove the comment above)?

Reply:
Done. Thank you for the commitment. Fixing is not a problem, but resolving conflicts when rebasing is a pain 😃.

Reply:
Yes, I've experienced this recently with my own PRs. There's too much activity in Julia at the moment -- which is actually a good sign!